### PR TITLE
[Alicloud BOSH Docs] Add Bosh Docs for Alibaba Cloud CPI

### PR DESCRIPTION
**Pull request by xiaozhu36**
This PR is the second submit and the first submit is #429 

The PR add bosh docs which applies to alibaba cloud CPI configuration and properties.
The PR includes the following files:

* init-alicloud.md: creating alicloud environment details before deploying BOSH
* alicloud-cpi.md: alicloud bosh CPI details
* alicloud-cpi-errors.md: alicloud bosh CPI common errors

### CODE REVIEW

File: content/build-cpi.md [hunk 1]
```diff
@@ -29,7 +29,9 @@ The [`bosh_cpi`](https://rubygems.org/gems/bosh_cpi) gem provides a `Bosh::Cpi::
 
 There are a few CPI releases written in Go, as well:
 
+ * [Alibaba Cloud CPI Release](https://github.com/cloudfoundry-incubator/bosh-alicloud-cpi-release)
  * [Google CPI Release](https://github.com/cloudfoundry/bosh-google-cpi-release)
+ * [Google CPI Release](https://github.com/cloudfoundry-incubator/bosh-google-cpi-release)
```
Comment (jfmyers9) [inline content/build-cpi.md]: This seems unrelated to the PR. Is there a reason for adding this or was this a mistake?

Comment (xiaozhu36) [reply content/build-cpi.md]: Mistake and fixed.
